eclass/office-ext-r1.eclass

```diff
@@ -142,9 +142,13 @@ office-ext-r1_src_install() {
 			for j in ${OFFICE_EXTENSIONS[@]}; do
 				pushd "${WORKDIR}/${j}/" > /dev/null || die
 				insinto /usr/$(get_libdir)/${i}/share/extensions/${j/.oxt/}
-				doins -r *
+				doins -r .
```

Comment:
These are not completely equivalent, when it comes to `.*` files. Probably not a problem, but have you checked?

Reply:
verified, no difference